serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/security/EntitySigner.java

```diff
@@ -0,0 +1,36 @@
+package no.difi.meldingsutveksling.serviceregistry.security;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import com.nimbusds.jose.*;
+import com.nimbusds.jose.crypto.RSASSASigner;
+import com.nimbusds.jose.util.Base64;
+import no.difi.meldingsutveksling.serviceregistry.controller.EntityResource;
+import no.difi.move.common.oauth.KeystoreHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.security.cert.CertificateEncodingException;
+import java.util.List;
+
+@Component
+public class EntitySigner {
+
+    @Autowired
+    private KeystoreHelper keystoreHelper;
+
+    public String sign(EntityResource entity) throws JsonProcessingException, CertificateEncodingException, JOSEException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: com.fasterxml.jackson.core.JsonProcessingException, java.security.cert.CertificateEncodingException, com.nimbusds.jose.JOSEException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)